src/ui/qgsprojectpropertiesbase.ui

```diff
@@ -605,7 +605,7 @@
                    <widget class="QLabel" name="textLabel1_8">
                     <property name="text">
                      <string>Ellipsoid
-(for distance and area calculations)</string>
+(distance, area, angle calculations)</string>
```

Comment:
```suggestion
(distance, area and angle calculations)</string>
```
?

Reply:
I tried not to make the text any longer.
Original:
![image](https://github.com/user-attachments/assets/a02935e9-2b23-487a-97ff-b28d7f2b2c29)
My PR
![image](https://github.com/user-attachments/assets/f4114624-903e-4d0c-a242-288fd1fb9d6b)
Suggested change:
![image](https://github.com/user-attachments/assets/0631488d-f8ba-4178-9b27-7e0700bb0e73)

Reply:
Remove the text in parenthesis and let's go for a tooltip?

Reply:
What do you think?
![image](https://github.com/user-attachments/assets/e3fb8ff5-91d1-4c9b-b884-89a2f34aa62b)
(The tooltip is only on the label, not on the combobox)

Reply:
perfect!

Reply:
PR updated.